File: bootcamp-2023/integration-testing/exercise/c#-dotnet-5/TripServiceKata/TripServiceKata/User/UserSession.cs
using TripServiceKata.Exception;

namespace TripServiceKata.User
{
    public class UserSession
    {
        private readonly User user;
        public UserSession(User user)
        {
            this.user = user;
        }

        public User? GetLoggedUser()
        {
            return user;
        }
    }
}

File: bootcamp-2023/integration-testing/exercise/php/src/TripServiceKata/User/UserSession.php
<?php

namespace TripServiceKata\User;

use TripServiceKata\Exception\DependentClassCalledDuringUnitTestException;

class UserSession
{
    private $user;

    public function __construct(User $user) {
        $this->user = $user;
    }

    public function getLoggedUser()
    {
        return $this->user;
    }

}
